pyproject.toml

```diff
@@ -40,6 +40,7 @@ dependencies = [
   "wrapt",
   "sortedcontainers",
   "protobuf>=3.20, <6.0",
+  "ddsketch",
```

Comment:
I thought we already had this?

Reply:
we had removed it previously because i had found a different way to compute quantiles